Dockerfile

```diff
@@ -6,8 +6,8 @@ ENV APIGENTOOLS_BASE_DIR=/var/lib/apigentools
 ENV APIGENTOOLS_SPEC_REPO_DIR=${APIGENTOOLS_BASE_DIR}/spec-repo \
     _APIGENTOOLS_GIT_HASH_FILE=${APIGENTOOLS_BASE_DIR}/git-hash
 
-ENV OPENAPI_GENERATOR_VERSION=4.2.2 \
-    PACKAGES="docker findutils git golang-googlecode-tools-goimports java npm patch python3 python3-pip unzip"
+ENV OPENAPI_GENERATOR_VERSION=4.2.3-SNAPSHOT \
```

Comment:
Is there a way to pin to a specific commit instead so that it doesn't change ?

Reply:
We could try to build it from source or maybe copy from docker image with a specific digest `sha256:...`.